src/devices/yokis.ts

```diff
@@ -0,0 +1,2642 @@
+            .withCategory('config');
+
+        const toZigbee: Tz.Converter[] = [
+            {
+                key: ['RelaunchBleAdvert'],
+                convertSet: async (entity, key, value, meta) => {
+                    yokisExtendChecks.log(key, value);
+                    await entity.command('manuSpecificYokisDevice', 'relaunchBleAdvert', {});
+                },
+            },
+        ];
+
+        return {
+            exposes: [exposes],
+            toZigbee,
+            isModernExtend: true,
+        };
+    },
+    openNetwork: (): ModernExtend => {
+        const exposes = e
+            .composite('OpenNetworkCommand', 'OpenNetworkProp', ea.SET)
+            .withDescription('Open ZigBee network')
+            .withFeature(
+                e
+                    .numeric('uc_OpeningTime', ea.SET)
+                    .withValueMin(0)
+                    .withValueMax(255)
+                    .withUnit('s')
+                    .withDescription('Opening time wanted from 1 to 255 seconds,0 means closing the network.'),
+            )
+            .withCategory('config');
+
+        const toZigbee: Tz.Converter[] = [
+            {
+                key: ['OpenNetworkProp'],
+                convertSet: async (entity, key, value, meta) => {
+                    const commandWrapper = yokisExtendChecks.parseOpenNetworkInput(value);
+
+                    yokisExtendChecks.log(key, value);
+
+                    await entity.command('manuSpecificYokisDevice', 'OpenNetwork', commandWrapper.payload);
+                },
+            },
+        ];
+
+        return {
+            exposes: [exposes],
+            toZigbee,
+            isModernExtend: true,
+        };
+    },
+    moveToPosition: (): ModernExtend => {
+        const exposes = e
+            .composite('move_to_position_command', 'moveToPositionProp', ea.SET)
+            .withDescription(
+                'Move to position specified in uc_BrightnessEnd parameter.' +
+                    'If TOR mode is set (no dimming) or MTR : if uc_BrightnessEnd under 50% will set to OFF else will be set to ON',
+            )
+            .withFeature(e.numeric('uc_BrightnessStart', ea.SET).withDescription(''))
+            .withFeature(e.numeric('uc_BrightnessEnd', ea.SET).withDescription(''))
+            .withFeature(
+                e
+                    .numeric('ul_PreTimerValue', ea.SET)
+                    .withUnit('s')
+                    .withDescription('If defined will force the pretimer value (only for this order) if not the device will use its own value.'),
+            )
+            .withFeature(
+                e
+                    .binary('b_PreTimerEnable', ea.SET, true, false)
+                    .withDescription('If defined will force the pretimer use (only for this order) if not the device will use its own value.'),
+            )
+            .withFeature(
+                e
+                    .numeric('ul_TimerValue', ea.SET)
```

Comment:
Please also use snake casing here, also, what is `ul`/`b` (it not very descriptive)

Reply:
It actually matches the attribute names from the manufacturer documentation, so i kept the names to ease the comparison with their specifications.

Reply:
Done